config_matrix.html

```diff
@@ -0,0 +1,166 @@
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+</head>
+
+<table class="table table-bordered">
+<tr>
+  <th></th><th>Report Direct to Compliance</th><th>Report Direct to Automate</th><th>Report to Compliance via Chef Server</th><th>Report to Automate via Chef Server</th>
+</tr>
+<tr>
+  <th>Fetch Direct From Compliance</th>
+  <td> <!-- Fetch Directly from Compliance : Report Directly to Compliance -->
+    <pre>
+    ['audit']['reporter'] = 'chef-compliance'
+    ['audit']['server'] = 'https://compliance-server.test/api'
+    ['audit']['refresh_token' OR 'token'] = '..'
+    ['audit']['owner'] = 'Owner/Org'
+    </pre>
+  </td>
+  <td> <!-- Fetch Directly from Compliance : Report Directly to Automate -->
+    <pre>
+    ['audit']['reporter'] = 'chef-automate'
+    ['audit']['server'] = 'https://compliance-server.test/api'
+    ['audit']['refresh_token' OR 'token'] = '..'
+    ['audit']['owner'] = 'Owner/Org'
+
+    client.rb:
+      data_collector['server_url'] = 'https://automate-server.test/data-collector/v0/'
+      data_collector['token'] = '..'
+    </pre>
+  </td>
+  <td> <!-- Fetch Directly from Compliance : Report to Compliance via Chef Server -->
+    <pre>
+    ['audit']['reporter'] = 'chef-server-compliance'
+    ['audit']['server'] = 'https://compliance-server.test/api'
+    ['audit']['refresh_token' OR 'token'] = '..'
+    ['audit']['owner'] = 'Owner/Org'
+    </pre>
+  </td>
+  <td> <!-- Fetch Directly from Compliance : Report to Automate via Chef Server -->
+    <pre>
+    ['audit']['reporter'] = 'chef-server-automate'
+    ['audit']['server'] = 'https://compliance-server.test/api'
+    ['audit']['refresh_token' OR 'token'] = '..'
+    ['audit']['owner'] = 'Owner/Org'
+
+    chef-server.rb:
+      data_collector['root_url'] = 'https://automate-server.test/data-collector/v0/'
+    </pre>
+  </td>
+</tr>
+<tr>
+  <th>Fetch From Compliance via Chef Server</th>
+  <td> <!-- Fetch from Compliance via Chef Server : Report Directly to Compliance -->
+    <pre>
+    ['audit']['reporter'] = 'chef-compliance'
+    ['audit']['fetcher'] = 'chef-server'
+    ['audit']['server'] = 'https://compliance-server.test/api'
+    ['audit']['refresh_token' OR 'token'] = '..'
+    ['audit']['owner'] = 'Owner/Org'
+
+    Compliance Integrated w/ Chef Server
+    </pre>
+  </td>
+  <td> <!-- Fetch from Compliance via Chef Server : Report Directly to Automate -->
+    <pre>
+    ['audit']['reporter'] = 'chef-automate'
```

Comment:
```
69   <td> <!-- Fetch from Compliance via Chef Server : Report Directly to Automate -->
   70     <pre>
   71     ['audit']['reporter'] = 'chef-automate'
   72     ['audit']['fetcher'] = 'chef-server'
   73     ['audit']['server'] = 'https://compliance-server.test/api'
   74     ['audit']['refresh_token' OR 'token'] = '..'
   75     ['audit']['owner'] = 'Owner/Org'
   76
   77     Compliance Integrated w/ Chef Server
   78
   79     client.rb:
   80       data_collector['server_url'] = 'https://automate-server.test/data-collector/v0/'
   81       data_collector['token'] = '..'
   82     </pre>
   83   </td>
```
Again, I think it's correct as-is.  use fetcher=chef-server to fetch via Integration, use report=chef-automate to report Directly to Automate, not via chef-server integration.